extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -412,14 +412,14 @@ oppia.directive('graphViz', [
             text: 'Labeled',
             option: 'isLabeled'
           },
-          {
-            text: 'Directed',
-            option: 'isDirected'
-          },
-          {
-            text: 'Weighted',
-            option: 'isWeighted'
-          }];
+                                 {
```

Comment:
Woah. What happened here?

Reply:
This is very weird! Looking into such indentations (I found a few above too). I am not too sure why this is occurring unless it is a false positive.

Reply:
@seanlip: I tried out various 4.X versions, namely 4.7.0, 4.18.0, 4.18.1, 4.18.2 but the `indent` rule seems to produce the same result. The `indent-legacy` rule works perfectly fine though. What should we do here then? 
Thanks!

Reply:
I have shifted { to the next line. This fixes the indentation (as suggested in Issue [#10106](https://github.com/eslint/eslint/issues/10106)) by the Eslint team.

Reply:
I read their response differently. I think you're missing some kind of config option. The following should be valid and we need to find a way to make it so.

```
$scope.graphOptions = [{
  text: 'Labeled',
  option: 'isLabeled'
},
{
  text: 'Directed',
  option: 'isDirected'
},
{
  text: 'Weighted',
  option: 'isWeighted'
}];
```

Also, the following should be valid (and is preferable due to compactness):

```
$scope.graphOptions = [{
  text: 'Labeled',
  option: 'isLabeled'
}, {
  text: 'Directed',
  option: 'isDirected'
}, {
  text: 'Weighted',
  option: 'isWeighted'
}];
```

So some linter changes are needed in that regard.

Reply:
Yes, you're right. There must be some way to get around that configuration. I'll look through the existing rules once. Thanks!